tools/gulp/tasks/payload.ts

```diff
@@ -0,0 +1,56 @@
+import {task} from 'gulp';
+import {join} from 'path';
+import {statSync, readFileSync} from 'fs';
+import {DIST_COMPONENTS_ROOT} from '../constants';
+import {openFirebaseDatabase} from '../task_helpers';
+import {spawnSync} from 'child_process';
+
+// Those imports lack types.
+const uglifyJs = require('uglify-js');
+
+const BUNDLE_PATH = join(DIST_COMPONENTS_ROOT, 'bundles', 'material.umd.js');
+
+/** Task which runs test against the size of whole library. */
+task('payload', ['build:release'], () => {
+
+  let results = {
+    normal: getFilesize(BUNDLE_PATH),
+    minified_uglify: getUglifiedSize(BUNDLE_PATH),
+    timestamp: Date.now()
+  };
+
+  // Print the results to the console, so we can read it from the CI.
+  console.log('Payload Results:', JSON.stringify(results, null, 2));
+
+  // Publish the results to firebase when it runs on Travis and not as a PR.
+  if (process.env['TRAVIS_PULL_REQUEST'] === 'false') {
```

Comment:
Created a method `isTravisPushBuild` because if we create `isTravisPullRequest`, the method will also return `false` when we not run in Travis at all. Using `isTravisPushBuild` should be fine for now.